tests/test.py

```diff
@@ -297,6 +297,10 @@ def test_sheet1(self):
     def test_get_worksheet(self):
         sheet1 = self.spreadsheet.get_worksheet(0)
         self.assertTrue(isinstance(sheet1, gspread.Worksheet))
+    
+    def test_get_worksheet_by_id(self):
+        sheet1 = self.spreadsheet.get_worksheet_by_id(0)
+        self.assertTrue(isinstance(sheet1, gspread.Worksheet))
```

Comment:
Same here, if you could please adjust the test to reflect what the users would be likely to do :
- add a couple of sheets (3 for example seems enough)
- get the second sheet (id = 1)

as mentioned above the first sheet is already access via a shortcut so most likely the test would not reflect the use case of that function.

thank you.